reform_test.py

```diff
@@ -0,0 +1,241 @@
+"""
```

Comment:
Seems like some reasonable tests here, although it seems to me that this is testing the same functionality for various parameters. The maintenance burden goes up for each hard coded value that is tested (since it could change in the future), but if you think there is benefit in keeping all of the functions here, no problem. The right place for this file is taxcalc/tests/test_reform.py (or really anything starting with `test_`, with each of the functions prefaced with `test_` so that py.test will run them each time.